featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -347,12 +348,18 @@ def test_diff(pd_es):
     )
     diff2 = ft.Feature(value, groupby=customer_id_feat, primitive=Diff)
 
-    feature_set = FeatureSet([diff1, diff2])
+    diff_periods = ft.Feature(
+        value, groupby=customer_id_feat, primitive=Diff(periods=1)
```

Comment:
I like how you split the diff datetime tests into two below -- could you pull out the periods checks into a second `test_diff_shift` test as well?